tabs/osd.js

```diff
@@ -340,10 +342,9 @@ OSD.updateDisplaySize = function() {
 OSD.msp = {
   encodeOther: function() {
     var result = [-1, OSD.data.video_system];
-    if (semver.gte(CONFIG.apiVersion, "1.21.0")) {
+    if (semver.gte(CONFIG.flightControllerVersion, "3.0.1")) {
```

Comment:
I did some work on this a while ago, mainly to try avoid some MSP clashes between Cleanflight, betaflight and iNav.

I'm not actively working on it at the moment.

Surely we should be using MSP versions, not FC versions.